### PR TITLE
Imported chapters from Deployment guide

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Review by cwickert — APPROVED** [review]
Delete the comment from book_administration.xml and feel free to merge. The other two comments I made are up to you.

File: xml/book_administration.xml [hunk 1]
```diff
@@ -84,17 +83,19 @@
   <xi:include href="suse_linux.xml"/>
   <xi:include href="nm.xml"/>
   <xi:include href="powermanagement.xml"/>
-<!-- FIXME cwickert 2018-10-26: A single chapter doesn't justify a part of
- it's own. However, we might want to have a 'Hardware' part once we have
- more content. For now, the part is commented out.
--->
```
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/book_administration.xml:90]: i thought i already did that ?!?

[the remark this answers is not in the document]
Comment (cwickert) [reply xml/book_administration.xml:90]: Oh, sorry, my bad.

**Comment by cwickert** [comment]
> once you fixed book_startup-opensuse.xml (needs the same changes as book_administration.xml).

It's probably easier to just keep yast2_hw.xml. It will be openSUSE-only, but you don't have to mess with the openSUSE docs then.

File: xml/book_administration.xml [hunk 2]
```diff
@@ -84,17 +83,19 @@
...
-  <xi:include href="hardware_vm_guest.xml"/>
+ <part xml:id="part-hardware">
+  <title>Hardware configuration</title>
+  <xi:include href="yast2_keymouse.xml"/>
+  <xi:include href="yast2_sound.xml"/>
+  <xi:include href="yast2_printer.xml"/>
+  <xi:include os="sled;osuse" href="yast2_scanner.xml"/>
+  <!-- 2022-03-29 tbazant: hot-plugging CPUs in VM is covered in the Virtualization
+   Guide, thus commenting out -->
+   <!-- <xi:include href="hardware_vm_guest.xml"/> -->
```
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/book_administration.xml]: working on it separately

File: xml/book_administration.xml [hunk 3]
```diff
@@ -84,17 +83,19 @@
   <xi:include href="suse_linux.xml"/>
   <xi:include href="nm.xml"/>
   <xi:include href="powermanagement.xml"/>
```
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/book_administration.xml]: moved

**Pull request merged**